app/controllers/checkins_controller.rb

```diff
@@ -0,0 +1,28 @@
+class CheckinsController < ApplicationController
+  def new
+    @checkin = Checkin.new
+    @contact_options = Contact.all.map{|u| [ u.company_name, u.id ] }
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space inside square brackets detected.